packages/flutter/lib/src/material/popup_menu.dart

```diff
@@ -38,6 +38,14 @@ const double _kMenuVerticalPadding = 8.0;
 const double _kMenuWidthStep = 56.0;
 const double _kMenuScreenPadding = 8.0;
 
+/// Used to configure how the [PopupMenuButton] positions its popup menu.
+enum MenuPosition {
```

Comment:
I know it's longer, but can we rename this to `PopupMenuPosition`?  I'm working on a `MenuBar` widget, and while I'm not using an enum (at least not yet), people might be confused as to which widget this works with if they didn't read the docs.

Reply:
That makes sense, renaming it later would be a breaking change, better safe now :)